rpc/sunrpcserver/dump_prog.go

```diff
@@ -0,0 +1,90 @@
+package sunrpcserver
+
+// GfDump is a type for GlusterFS Dump RPC program
+type GfDump genericProgram
+
+func newGfDump() *GfDump {
+	// rpc/rpc-lib/src/xdr-common.h
+	return &GfDump{
+		name:        "GF-DUMP",
+		progNum:     uint32(123451501),
+		progVersion: uint32(1),
+		procedures: []Procedure{
+			Procedure{uint32(1), "Dump"}, // GF_DUMP_DUMP
```

Comment:
Same with the procnums. Define consts and use them here instead of the number directly.